### DISCUSSION TITLE
Installing on shared hosting server?

### DISCUSSION THREAD

**Discussion by senadaruc** [outline/outline#4374] [General]
Hello,

Is there a manual for installing on a shared hosting web server?

Thanks,

**Comment by tommoor** [comment]
Documentation for self hosing is here:
https://wiki.generaloutline.com/s/770a97da-13e5-401e-9f8a-37949c19f97e

**Comment by senadaruc** [comment]
Thanks, but those solutions are not possible on a shared web host like I install WordPress!

**Comment by Limezy** [comment]
You won't be able to install Outline for sure. Shared web server are usually giving you a basic outdated php and sql database. Outline requires much more than that !
